app/interactors/users/checkins/import_checkins.rb

```diff
@@ -14,6 +15,10 @@ def call
 
     private
 
+    def creat_activity
+      device.create_activity :import, owner: device.user, parameters: { count: CSV.read(path).length }
```

Comment:
Yes, it looks like the same.

But actually this will be one interface, one entry point.
As a result: once you create interactor correctly, the case of double mistyped `creat_activity` will no longer possible.